kornia/augmentation/container/patch.py

```diff
@@ -338,19 +338,14 @@ def apply_by_param(
         if label is not None and out_label is not None:
             if len(out_label.shape) == 1:
                 # Wierd the mypy error though it is as same as in the next block
-                _label = (
-                    torch.ones(  # type: ignore
-                        in_shape[0] * in_shape[1], device=out_label.device, out_label=label.dtype
-                    )
-                    * -1
-                )
+                _label = torch.ones(in_shape[0] * in_shape[1], device=out_label.device, dtype=out_label.dtype) * -1
```

Comment:
```suggestion
                _label = torch.full(in_shape[0] * in_shape[1], -1, device=out_label.device, dtype=out_label.dtype)
```
we might want to start using `torch.full` for such cases to avoid matrix multiplication
https://pytorch.org/docs/stable/generated/torch.full.html#torch.full